react-1/src/components/User/User.js

```diff
@@ -0,0 +1,77 @@
+import React, { useImperativeHandle, useRef } from "react";
+import { useStyles } from "./style";
+import { DragSource, DropTarget } from "react-dnd";
```

Comment:
Those are the legacy API. The new API for the hooks are using `useDrag` and `useDrop` effects.